### PR TITLE
Lower Trilu in opset 14 to krnl

### PR DESCRIPTION
**Pull request by tungld**
This patch lowers ONNXTrilu into Krnl.

ONNXTrilu is like an unary operation, but its computation uses loop indices that has not supported yet by the lowering pattern of unary operations. So, this patch lowers ONNXTrilu as an independent operation.

### CODE REVIEW

File: src/Conversion/ONNXToKrnl/Math/Trilu.cpp [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    create.krnl.iterateIE(loopDef, loopDef, lbs, ubs,
+        [&](KrnlBuilder &createKrnl, ValueRange loopInd) {
+          MultiDialectBuilder<KrnlBuilder, MathBuilder, SCFBuilder> create(
+              createKrnl);
+          Value i = create.math.add(k, loopInd[rank - 2]);
+          Value j = loopInd[rank - 1];
+          Value retainCond =
+              retainUpper ? create.math.gt(i, j) : create.math.lt(i, j);
+
+          create.scf.ifThenElse(
```
[the remark this answers is not in the document]
Comment (tungld) [reply src/Conversion/ONNXToKrnl/Math/Trilu.cpp:76]: Yes, it is possible, I will change to `select`

[the remark this answers is not in the document]
Comment (tungld) [reply src/Conversion/ONNXToKrnl/Math/Trilu.cpp]: @AlexandreEichenberger I changed to use `select`, it works well. Thanks for your suggestion!

**Comment by AlexandreEichenberger** [comment]
@gongsu832 is it possible that the linux x86 is down?

**Review by AlexandreEichenberger — APPROVED** [review]
LGTM, tx

**Comment by AlexandreEichenberger** [comment]
Let us know if you believe that Triu is key to performance and you believe it should be simdized

**Pull request merged**